CHANGELOG.md

```diff
@@ -1 +0,0 @@
-Release notes are now available in our documentation at ([elastic.co](https://www.elastic.co/guide/en/apm/agent/rum-js/current/release-notes.html))
```

Comment:
we have to update our releases to point to release notes instead of changelog.md